jdaviz/app.py

```diff
@@ -2759,9 +2759,13 @@ def compose_viewer_area(viewer_area_items):
             # plugin itself
             tray_item_label = tray.get('label')
 
-            config_name = config['settings']['configuration']
-            tray_item_description = get_plugin_description(config_name,
-                                                           tray_item_label)
+            if 'configuration' in config['settings'].keys():
+                config_name = config['settings']['configuration']
```

Comment:
there should be a `self.config` which would be set to "unknown" if it doesn't exist.